src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/FieldMakerTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.unittest.parser;
+
+import com.alibaba.druid.sql.ast.expr.SQLIntegerExpr;
+import com.alibaba.druid.sql.ast.expr.SQLNumberExpr;
+import com.amazon.opendistroforelasticsearch.sql.domain.MethodField;
+import com.amazon.opendistroforelasticsearch.sql.exception.SqlParseException;
+import com.amazon.opendistroforelasticsearch.sql.parser.FieldMaker;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class FieldMakerTest {
+
+    private static final String ALIAS = "a";
+
+    private static final String TABLE_ALIAS = "t";
+
+    @Test
+    public void makeFiled_assign() throws SqlParseException {
```

Comment:
Typo: I believe you meant ```makeField_assign()```, same for the test below

Reply:
Just to add, we should be consistent with Java naming convention. It should be `makeFieldAssign()` here.

Reply:
Done.